homeassistant/components/http.py

```diff
@@ -185,6 +195,41 @@ def register_path(self, method, url, callback, require_auth=True):
         """ Regitsters a path wit the server. """
         self.paths.append((method, url, callback, require_auth))
 
+    def check_expired_sessions(self):
```

Comment:
I think the name `remove_expired_sessions` better explains what this method does